index.js

```diff
@@ -109,15 +77,16 @@ function FastForEach(spec) {
   this.afterAdd = spec.afterAdd;
   this.beforeRemove = spec.beforeRemove;
   this.templateNode = makeTemplateNode(
-    spec.name ? document.getElementById(spec.name).cloneNode(true) : spec.element
+    spec.templateNode || (spec.name ? document.getElementById(spec.name).cloneNode(true) : spec.element)
```

Comment:
Thanks @cervengoc – I just wanted to check that I wasn't missing something.  No worries on being short of time —— tis the season :santa: 

What I'm thinking of doing is having the `template` argument take either a string (DOM node identifier) or a node, an array of nodes, or a DOM `template` node.  I'm still not sure if that's a good idea. :)